--- src/hardware/r_opengl/r_opengl.c.old	2020-05-11 23:31:38.000000000 -0400
+++ src/hardware/r_opengl/r_opengl.c	2020-05-20 15:40:58.466898373 -0400
@@ -142,13 +142,13 @@
 // Returns          :
 // -----------------+
 
-#ifdef DEBUG_TO_FILE
+#if 0
 FILE *gllogstream;
 #endif
 
 FUNCPRINTF void GL_DBG_Printf(const char *format, ...)
 {
-#ifdef DEBUG_TO_FILE
+#if 0
 	char str[4096] = "";
 	va_list arglist;
